modules/core/src/main/java/org/apache/ignite/internal/processors/affinity/GridAffinityAssignmentCache.java

```diff
@@ -76,23 +76,23 @@ public class GridAffinityAssignmentCache {
      * Affinity cache will shrink when total number of non-shallow (see {@link HistoryAffinityAssignmentImpl})
      * historical instances will be greater than value of this constant.
      */
-    private final int MAX_NON_SHALLOW_HIST_SIZE = getInteger(IGNITE_AFFINITY_HISTORY_SIZE, DFLT_AFFINITY_HISTORY_SIZE);
+    final int maxNonShallowHistSize = getInteger(IGNITE_AFFINITY_HISTORY_SIZE, DFLT_AFFINITY_HISTORY_SIZE);
```

Comment:
Let's revert this and rename in another PR.

Reply:
It's a minor refactoring, I think creating a new ticket for this - it's a waste of resources

Reply:
Right now it's very hard to review changes because of useless renaming.
Can you, please, for the sake of reviewer revert it.